test/integration/platform_admin_can_only_delete_cars_test.rb

```diff
@@ -7,15 +7,17 @@ class PlatformAdminCanOnlyDeleteCarsTest < ActionDispatch::IntegrationTest
 
     visit cars_path
 
-    assert page.has_content?('Dave\'s Cars')
+    click_link "View Car"
+
     assert page.has_content?(100)
     assert page.has_content?('El Camino')
-    assert page.has_content?('Daily Price')
+    assert page.has_content?('Daily price:')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.